VimeoNetworking/VimeoNetworking/Models/VIMSeason.h

```diff
@@ -17,4 +17,11 @@
 - (nullable VIMConnection *)connectionWithName:(nonnull NSString *)connectionName;
 - (nullable VIMInteraction *)interactionWithName:(nonnull NSString *)name;
 
+/**
+ Reports whether this season is protected by DRM.
+
+ @return A BOOL indicating whether the receiver is a DRM-protected season.
+ */
+- (BOOL)isDRMProtected;
```

Comment:
Again maybe we can make this more descriptive. Since the content is DRM protected rather than the season object itself, maybe `containsDRMProtectedContent` or something